apps/ai-image-tagging/lambda/usage.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+const { promisify } = require('util');
+
+const AWS = require('aws-sdk');
+
+module.exports = async spaceId => {
+  const ddb = new AWS.DynamoDB();
+  const updateItem = promisify(ddb.updateItem.bind(ddb));
```

Comment:
I'd suggest using DynamoDB DocumentClient (https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/DynamoDB/DocumentClient.html) as it does DynamoDB deserialization for you

The AWS SDK also supports promises with `.promise()` on the result of a request (https://docs.aws.amazon.com/sdk-for-javascript/v2/developer-guide/using-promises.html) so you can get rid of `promisify`